src/components/AnswerCard.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { FiArrowUpRight } from 'react-icons/fi'
+import styled from "styled-components";
+import graphql from 'babel-plugin-relay/macro'
+import { createFragmentContainer } from 'react-relay'
+
+import { Text } from '../styles'
+
+function AnswerCard(props) {
+  const { id,
+    answer,
+    text,
+    citationUrl,
+    citationTitle,
+    credibleCount,
+    notCredibleCount } = props.answer
+  const setColor = (answer === 'True') ? true : false;
+
+  return <div key={id}>
+    <>
+      <AnswerHeader children={answer} setColor={setColor} />
+    </>
+    <>
+      <Text.Small>Asked by <b>DEMO</b> </Text.Small>
+    </>
+    <>
+      <Text.Small children={text} />
+    </>
+    <MediaWrapper>
+      <FiArrowUpRight />
+      <MediaLink onClick={event => { event.stopPropagation(); }} href={citationUrl} >{citationUrl}</MediaLink>
+      <MediaTitle>{citationTitle}</MediaTitle>
+    </MediaWrapper>
+    <ButtonWrapper>
+      <VoteButton background={`background: #23BE7B;`}>
```

Comment:
There's not need to write background twice, you can simply pass the color and output the `background` string in the styled component. But I think you should have a global colors file anyway. Don't write hex values inline.